src/checker.js

```diff
@@ -69,69 +69,130 @@ class Checker {
         return false
     }
 
-    static isValidFormat(value,expression) {
+    static isValidFormat(value, expression) {
         return expression.test(value)
     }
-    
+
+    static isValidURL(value, options) {
+
+        var schemes = options.schemes || this.schemes || ['http', 'https']
```

Comment:
I seems `this.schemes`, `this.allowDataUrl` or `this.allowLocal` will never be true or have value